code/js/controllers/SpotifyController.js

```diff
@@ -24,7 +24,7 @@
     var that = this;
     var selectorIndex;
 
-    if (window.location.hostname === "open.spotify.com") {
+    if (window.location.hostname === "open.spotify.com" || window.location.hostname === "play.spotify.com") {
```

Comment:
Isn't this going to make it always use `selectorIndex = 2` then? AFAIK these are the only two possible URLs for the Spotify player

Also, if you look at the player markup here: https://github.com/berrberr/streamkeys/issues/286#issuecomment-285902581 I do not see any element with `#play-pause` -- where are you getting that from?

I think that we need separate play and pause selectors of `. spoticon-play-32` , `. spoticon-pause-32`